rpc/types/src/json.rs

```diff
@@ -1534,6 +1552,203 @@ define_request_and_response! {
     }
 }
 
+define_request_and_response! {
+    UNDOCUMENTED_METHOD,
+    cc73fe71162d564ffda8e549b79a350bca53c454 =>
+    core_rpc_server_commands_defs.h => 2798..=2823,
+
+    GetTxIdsLoose,
+
```

Comment:
This is going to be an annoying endpoint to support, it may require DB changes, depending on what guarantees we need to make

Reply:
I would probably be ok dropping support for this end point.